easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '6.1.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.4'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/%(namelower)s/%(namelower)s-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+]
+
+sources = [
+    'gcc-%(version)s.tar.bz2',
+    'gmp-6.1.1.tar.bz2',
+    'mpfr-%s.tar.bz2' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.16.1.tar.bz2',
+]
+
+builddependencies = [
+    ('M4', '1.4.17'),
+    ('binutils', '2.27'),
+]
+
+patches = [('mpfr-%s-allpatches-20160804.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
+
+checksums = [
+    '8fb6cb98b8459f5863328380fbf06bd1',     # gcc-6.1.0.tar.bz2
+    '4c175f86e11eb32d8bf9872ca3a8e11d',     # gmp-6.1.1.tar.bz2
+    'b8a2f6b0e68bef46e53da2ac439e1cf4',     # mpfr-3.1.4.tar.gz
+    'd6a1d5f8ddea3abd2cc3e98f58352d26',     # mpc-1.0.3.tar.gz
+    'ac1f25a0677912952718a51f5bc20f32',     # isl-0.16.1.tar.bz2
+    '13a670dca515e905856f95d13697fcdd',     # mpfr-3.1.2-allpatches-20160522.patch
+]
+
+languages = ['c', 'c++', 'fortran']
+
+withisl = False
```

Comment:
We have experienced similar issues with 'Could not find a usable isl' when building GCC 6.1.0 with the field withisl set to True. A hand build of GCC 6.1.0 with ISL support seems okay though.